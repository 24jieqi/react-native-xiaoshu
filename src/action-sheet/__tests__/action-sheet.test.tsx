import React from 'react'
// import type { ViewStyle } from 'react-native'
// import { StyleSheet, Text } from 'react-native'
import { create } from 'react-test-renderer'
// import type { ReactTestInstance } from 'react-test-renderer'

import { customRender } from '../../__tests__/test-utils'
import ActionSheet from '../action-sheet'

describe('ActionSheet', () => {
  it('render snapshot', () => {
    const tree = create(<ActionSheet visible actions={[]} />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('element', () => {
    const { queryByText } = customRender(
      <ActionSheet
        visible
        actions={[
          {
            name: '一个',
          },
        ]}
      />,
    )

    expect(queryByText('一个')).not.toBeNull()
  })
})
