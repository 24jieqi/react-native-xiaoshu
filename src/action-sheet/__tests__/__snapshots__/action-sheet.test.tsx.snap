// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionSheet render snapshot 1`] = `null`;
